harness/determined/common/experimental/determined.py

```diff
@@ -582,3 +583,48 @@ def stream_trials_validation_metrics(
             stacklevel=2,
         )
         return trial._stream_validation_metrics(self._session, trial_ids)
+
+    def describe_token(self, token_id: int) -> token.AccessToken:
+        """
+        Get the :class:`~determined.experimental.Token` representing the
+        token info with the provided token ID.
+        """
+        resp = bindings.get_GetAccessTokens(
+            session=self._session, tokenIds=[token_id], showInactive=True
+        )
+        return token.AccessToken._from_bindings(resp.tokenInfo, self._session)
+
+    def describe_tokens(self, token_ids: List[int]) -> token.AccessToken:
+        """
+        Get the :class:`~determined.experimental.Token` representing list of
+        token info with the provided token IDs.
+        """
+        resp = bindings.get_GetAccessTokens(
+            session=self._session, tokenIds=token_ids, showInactive=False
+        )
+        return token.AccessToken._from_bindings(resp.tokenInfo, self._session)
+
+    def list_tokens(
+        self, username: Optional[str] = None, show_inactive: Optional[bool] = None
+    ) -> token.AccessToken:
+        """
+        Get the :class:`~determined.experimental.Token` representing list of
+        token info with the provided username.
+        """
+        resp = bindings.get_GetAccessTokens(
+            session=self._session, username=username, showInactive=show_inactive
+        )
+        return token.AccessToken._from_bindings(resp.tokenInfo, self._session)
+
+    def create_token(
+        self, user_id: int, lifespan: Optional[str] = None, description: Optional[str] = None
```

Comment:
let's make `user_id` username here, since the other methods in the SDK use that as the standard identifier.

Reply:
also, we've decided to accept days, so let's just make it `expiration_days` here.